app/src/main/java/com/hedvig/app/feature/offer/ui/checkout/CheckoutActivity.kt

```diff
@@ -36,6 +38,7 @@ class CheckoutActivity : BaseActivity(R.layout.activity_checkout) {
     }
     private val viewModel: CheckoutViewModel by viewModel { parametersOf(parameter.quoteIds) }
     private val binding by viewBinding(ActivityCheckoutBinding::bind)
+    private lateinit var progressDialog: AlertDialog
```

Comment:
How come we're making this `lateinit`? 🤔 Couldn't we create this where it's used?

Reply:
Unfortunately not, we need the activity to be created before we can create the dialog

Reply:
I mean, can't we create it where we invoke it? That must be after `onCreate`, right? 🤔

Reply:
We want to be able to dismiss the dialog if we receive an error, thats why I created it here and then just do show/dismiss when handling an event

Reply:
Alright, fair enough.